Zajecia/src/main/java/pl/isa/javasmugglers/DatabaseWriter.java

```diff
@@ -41,5 +42,29 @@ public void addUserToDatabase(String userType, String userFirstName, String user
             e.printStackTrace();
         }
     }
+// kod testowy do wyświetlania odpowiednich użytkowników
```

Comment:
Przy mergowaniu do developa wyrzucamy kod testowy w komentarzach. Jeżeli chcesz to zachować, może stworzyć sobie jakąś utilową klasę, która Ci generuje testowych użytkowników. Teraz jak będziesz tego potrzebował za miesiąc, a coś w tym się zmieni, to ten kod i tak będzie przestarzały, bo nikt nie będzie wiedział, że trzeba było go zaktualizować, a tak klasa utils'owa zaczęłaby od razu błędami rzucać, nawet jak nie będzie używana ;)

Reply:
Jasne, usuwam. :)